packages/babel-plugin-transform-es3-reserved-words/src/reserved-words.js

```diff
@@ -0,0 +1,96 @@
+const es2017 = [
+  "await",
+  "break",
+  "case",
+  "catch",
+  "class",
+  "const",
+  "continue",
+  "debugger",
+  "default",
+  "delete",
+  "do",
+  "else",
+  "export",
+  "extends",
+  "finally",
+  "for",
+  "function",
+  "if",
+  "import",
+  "in",
+  "instanceof",
+  "new",
+  "return",
+  "super",
+  "switch",
+  "this",
+  "throw",
+  "try",
+  "typeof",
+  "var",
+  "void",
+  "while",
+  "with",
+  "yield",
+];
+const es3 = [
```

Comment:
could maybe `es2017` array be constructed by copying the `es3` + adding/removing neccessary items? such derived list is really helpful as documentation and its really easy to spot immediately what has changed between `es3` and `2017`

also I would sort `es3` array alphabetically (`es2017` is already sorted)

Reply:
Alphabetically sorted `es3` ✅ 

Currently left it as two arrays because I'm still not sure if these lists are not already available. If they stay, I agree that we should have them build upon each other 👍